pgjdbc/src/test/java/org/postgresql/test/jdbc2/UpdateableResultTest.java

```diff
@@ -529,4 +530,27 @@ public void testMultiColumnUpdate() throws Exception {
     st.close();
   }
 
+  @Test
+  public void testMultipleSchemas() throws Exception {
+    PgConnection pgConnection = con.unwrap(PgConnection.class);
+
+    String previousSchema = pgConnection.getSchema();
+
+    TestUtil.createSchema(con, "schema_update_row");
+    pgConnection.setSchema("schema_update_row");
+    TestUtil.createTable(con, "second",
```

Comment:
Please add a comment that this test relies on a definition of `second` table in another schema.
It is extremely hard to tell what does this test verify without looking into the PR.